charts/registry/templates/keycloak/database-credentials.yaml

```
@@ -22,7 +22,7 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: keycloak-database-credentials
+  name: {{ include "chart.fullname" . }}-keycloak-sec
```

I would choose a more descriptive name. E.g.:
```suggestion
  name: {{ include "chart.fullname" . }}-keycloak-db-creds
```

If you change this, please update also the references to this secret (which I couldn't find 🤔 - how is this secret been used?).